apps/climatemappedafrica/src/components/ExplorePage/index.js

```diff
@@ -134,6 +133,8 @@ function ExplorePage({ panelProps, profile: profileProp, ...props }) {
             onClick={handleClickMap}
             onClickUnpin={handleClickUnpin}
             zoom={7}
+            primaryGeoStyles={primaryGeoStyles}
+            secondaryGeoStyles={secondaryGeoStyles}
```

Comment:
These two styles needs to be passed in ... otherwise, how would one customise the map styling?